### DISCUSSION TITLE
Our next release - goodbye desktop builder 😢 hello web based builder 🤩

### DISCUSSION THREAD

**Discussion by mjashanks** [Budibase/budibase#1487] [Announcements]
From our next release forward, you will need to run Budibase in your self-hosted cloud. We will cease development on our old friend, the Budibase desktop builder (built on Electron). The Budibase builder will be served by your self-hosted deployment through the browser. 

Electron served us well, but the cloud is where our future lies for a number of reasons:

- Flexibility - users can now access their Budibase account on multiple devices.

- Collaboration - this is our first major step towards multi-user collaboration.

- Faster releases - developing on Electron brought with it additional work and effort that slowed us down. When developing in the cloud, we can ship new features quicker.

### Our next release
We intend to release the new Budibase on the 25th of May 2021. As well as the new cloud-based architecture, this release is packed full of features and improvements - stay tuned for full release notes.

To make use of all of these features, you will need to deploy Budibase using docker or Digital Ocean. Over time, we will continue to improve the self-hosting experience and offer our own cloud service.

### What is going to happen to old Budibase and my apps
The old Budibase will remain as it is today. We will no longer develop and enhance it. 
Old apps developed in the old Budibase (Electron) can be exported and then imported to your new Budibase instance in your self-hosted cloud environment.

As always, don’t hesitate to come and talk to us. You can comment here, or come to our [Discord server](https://discord.com/invite/rCYayfe).

Thanks,

Mike

**Comment by Florian-Schoenherr** [comment]
The people over at tauri would have even more reasons for you 😄
